src/experimentConfig.ts

```diff
@@ -29,7 +29,7 @@ const experimentConfigDisklet = makeReactNativeDisklet()
 // The probability of an experiment config feature being set for a given key
 const experimentDistribution = {
   swipeLastUsp: [50, 50],
-  createAccountType: [50, 50],
+  createAccountType: [100],
```

Comment:
This isn't a necessary change since we're not using generateExperimentConfigVal anymore for the cleaner. So optionally revert this line.